src/themes/_colors.css

```diff
@@ -0,0 +1,41 @@
+@layer base {
+    :root {
+        --primary: 173 6 6;
```

Comment:
there are accessibility issues regarding our primary/primary-dark/error colours against the standard black text, as the colour contrast ratios are not high enough to meet AA guidelines https://webaim.org/resources/contrastchecker/

Reply:
Huh, accessibility checker is now built in, that's an improvement. And here we are - one set of now AAA rated colour palettes. I'm expecting these to get tweaked by someone more designery as we go.

![image](https://github.com/bathlarp/frontend/assets/7863997/2835d489-5fd1-42a3-9d5b-abfea2483601)
![image](https://github.com/bathlarp/frontend/assets/7863997/31f2b58c-c34d-4280-875a-57e612af1821)
![image](https://github.com/bathlarp/frontend/assets/7863997/5e69d1d0-25e6-4aaa-b488-04e9aaea95a0)